tests/test_experiment.py

```diff
@@ -319,15 +352,22 @@ def test_megaquery_not_detectably_malformed():
     sql_lint(enrollments_sql)
 
     metrics_sql = exp.build_metrics_query(
-        metric_list=[m for m in mad.__dict__.values() if isinstance(m, Metric)],
+        metric_list=desktop_metrics,
         time_limits=tl,
         enrollments_table="enrollments",
     )
 
     sql_lint(metrics_sql)
 
 
-def test_segments_megaquery_not_detectably_malformed():
+def test_segments_megaquery_not_detectably_malformed(desktop_metrics):
+    desktop_segments = [
+        ConfigLoader.get_segment("regular_users_v3", "firefox_desktop"),
+        ConfigLoader.get_segment("new_or_resurrected_v3", "firefox_desktop"),
+        ConfigLoader.get_segment("weekday_regular_v1", "firefox_desktop"),
+        ConfigLoader.get_segment("allweek_regular_v1", "firefox_desktop"),
+        ConfigLoader.get_segment("new_unique_profiles", "firefox_desktop"),
+    ]
```

Comment:
Same question here: can we define test segments in that single file?

Reply:
Added in helpers/config_loader_lists.py